changelogs/head.asciidoc

```diff
@@ -13,6 +13,7 @@ https://github.com/elastic/apm-server/compare/6.5\...master[View commits]
 - Use _doc as document type for Elasticsearch >= 7.0.0 https://github.com/elastic/beats/pull/9056[beats/9057].
 - Automatically cap signed integers to 63bits https://github.com/elastic/beats/pull/8991[beats/8991].
 - Add optional `span.subtype` and `span.action` properties. {pull}1618[1618].
+- Build X-Pack binaries. {pull}1645[1645].
```

Comment:
@graphaelli do you this this is appropriate?

Reply:
I know what you mean, I can't think of anything better.  I'm asking around but I think you're good to merge and we can update the wording later before release so it makes sense to users.

Reply:
`Build default distribution under the Elastic License`